### PR TITLE
minor preview 10 post cleanup

### PR DESCRIPTION
**Pull request by sreuland**
while doing preview 10 updates, noticed two aspects to resolve:
* `initialize standalone` was not resulting in typescript generated code having the correct rpc url
* removed convert.ts, no longer needed, SorobanClient provides conversion scval conversions now

Closes https://github.com/stellar/soroban-example-dapp/issues/126

### CODE REVIEW

File: initialize.sh [hunk 1]
```diff
@@ -25,15 +25,19 @@ if [[ "$SOROBAN_RPC_HOST" == "" ]]; then
...
     SOROBAN_RPC_HOST="http://stellar:8000"
+    SOROBAN_RPC_URL="$SOROBAN_RPC_HOST"
   elif [[ "$NETWORK" == "futurenet" ]]; then
     SOROBAN_RPC_HOST="https://rpc-futurenet.stellar.org:443"
+    SOROBAN_RPC_URL="$SOROBAN_RPC_HOST"
   else
-    SOROBAN_RPC_HOST="http://localhost:8000"
+     # assumes standalone on quickstart, which has the soroban/rpc path
+    SOROBAN_RPC_HOST="http://localhost:8000/"
+    SOROBAN_RPC_URL="$SOROBAN_RPC_HOST/soroban/rpc"
```
Comment (chadoh) [inline initialize.sh:35]: Probably doesn't matter functionally, but there's a trailing slash in HOST and a leading slash here, so a double slash in the final

**Review by chadoh — APPROVED** [review]
One nit, but otherwise looks great. Thanks!

File: initialize.sh (hunk 1 above)
Comment (sreuland) [reply initialize.sh:35]: ah, good catch, thanks, fixed it.

**Review by Julian-dev28 — APPROVED** [review]
LGTM

**Pull request merged**